src/main/java/lotto/Money.java

```diff
@@ -0,0 +1,23 @@
+package lotto;
+
+import java.util.List;
+import static lotto.Validate.validate_Money;
+
+public class Money {
+
+    private static int money = 0;
+    private static final int lottoPrice = 1000;
```

Comment:
static field들의 네이밍 컨벤션은 UPPER_SNAKE_CASE입니다!
```suggestion
    private static final int LOTTO_PRICE = 1000;
```